src/main/js/tansaction/transaction-conf-modal.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Bank confirmation modal component.
+ *
+ * @author Xiaobing Hou
+ */
+import React from "react";
+import { Button, Modal } from "react-bootstrap";
+
+const TransactionConfModal = ({
+  selectedTransaction,
+  handleDelTransactions,
+  ...props
+}) => {
+  return (
+    <Modal
+      {...props}
+      size="md"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered
+    >
+      <Modal.Header closeButton>
+        <Modal.Title id="contained-modal-title-vcenter">
+          Bank account Id:{" "}
```

Comment:
Can we only have the transaction id in the header?

Reply:
done